src/scripts/pages/news/news.less

```diff
@@ -12,8 +12,88 @@
     .club-news {
         padding: 2rem;
         background-color: rgb(246, 246, 246);
+        width: 100%;
         text-align: center;
 
+        time {
+            display: block;
+        }
+
+        .club-news-time time {
+            color: #545454;
+            margin-bottom: 80px;
+        }
+
+        section {
+            position: relative;
+            margin: 2rem 1rem;
+            padding: 3rem;
+            background-color: white;
+            box-shadow: 0 0.1rem 0.2rem rgba(0, 0, 0, 0.1);
+
+            time {
+                padding: 10px;
+            }
+
+            h1, h2 {
+                text-align: center;
+            }
+
+            .subtitle {
+                font-style: italic;
+                color: #545454;
```

Comment:
Change this to use the site-wide color variable `@gray` (which is #555, quite close).